charts/thoras/values.yaml

```diff
@@ -53,4 +53,15 @@ thorasDashboard:
     memory: 100Mi
   port: 80
 
+thorasMonitor:
+  enabled: false
+  monitorCadence: "15"
+  maxJobLife: "600"
```

Comment:
Same deal as https://github.com/thoras-ai/helm-charts/pull/19/files#r1716050611 -- `maxJobLifeSeconds` would solve any confusion about what to input